pymtl3/passes/test/VcdGenPass_test.py

```diff
@@ -0,0 +1,62 @@
+#=========================================================================
+# VcdGenPass_test.py
```

Comment:
I guess we probably should remove this file because it was meant to be a toy/starting point to the development, and not a well-formed test case

Reply:
Can you also take a look at CollectSignalPass_test? It follows the style of this VcdGenPass, and I not sure if it is sufficient.

Reply:
I think passes like VcdGen and WaveGen are hard to test partly due to the lack of accurate reference output. The "toy" case itself is fine because it works as a simple directed test if we are able to somehow check the results of `SignalCollectPass` or `PrintWavePass`. Maybe asserting the names of signals appear in the output?

Reply:
Yeap, that works